includes/admin/class-sensei-onboarding.php

```diff
@@ -36,6 +36,12 @@ public function __construct() {
 
 		if ( is_admin() ) {
 			add_action( 'admin_menu', [ $this, 'admin_menu' ], 20 );
+			add_action( 'current_screen', [ $this, 'add_onboarding_help_tab' ] );
+
+			if ( $this->should_prevent_woocommerce_help_tab() ) {
+				// Prevent WooCommerce help tab.
+				add_filter( 'woocommerce_enable_admin_help_tab', '__return_false' );
```

Comment:
I wonder if we should instead have a method `should_enable_woocommerce_help_tab` and do the following:

`add_filter( 'woocommerce_enable_admin_help_tab', [ $this, 'should_enable_woocommerce_help_tab' ] );`

Rather than having a conditional and `__return_false`?

Reply:
Yes! Looks better! https://github.com/Automattic/sensei/commit/ec68ecb55d5a3c8f556d70ac7766b2f422d2fbf0